variant-remapping-automation/prepare_remapping_table.py

```diff
@@ -0,0 +1,294 @@
+import argparse
+import glob
+import gzip
+from urllib.parse import urlsplit
+
+import psycopg2
+from ebi_eva_common_pyutils.config_utils import get_accession_pg_creds_for_profile
+from ebi_eva_common_pyutils.logger import logging_config
+from ebi_eva_common_pyutils.metadata_utils import get_metadata_connection_handle
+from ebi_eva_common_pyutils.pg_utils import get_all_results_for_query, execute_query
+from ebi_eva_common_pyutils.taxonomy.taxonomy import get_scientific_name_from_ensembl
+
+from genome_target_tracker import get_tax_latest_asm_from_eva
+
+logger = logging_config.get_logger(__name__)
+
+
+def prepare_remapping_table(private_config_xml_file, remapping_version, release_version, noah_prj_dir, codon_prj_dir,
+                            profile):
+    scientific_names_from_release_table = get_scientific_name_from_release_table(args.private_config_xml_file)
+    # get a dictionary of taxonomy and the latest assembly it supports
+    tax_latest_support_asm = get_tax_latest_asm_from_eva(private_config_xml_file)
```

Comment:
I noticed another error across the script that we're using the taxonomy associated with the target assembly rather than the taxonomy associated with the project or study. 
Let's take an example: 
Project [PRJEB31574](https://www.ebi.ac.uk/eva/?eva-study=PRJEB31574) come from taxonomy [9915](http://www.ebi.ac.uk/ena/browser/view/Taxon:9915) (Zebu) and is mapped to [GCA_000003055.3](https://www.ebi.ac.uk/ena/browser/view/GCA_000003055.3) (Cow). When we're remapping it we should find the assembly supported for Zebu and not Cow which should be GCA_000247795.2 not GCA_002263795.2.

Reply:
updated to get the taxonomy of a project from **project_taxonomy** table.
There is a parameter called **taxonomyAccession** in the **batch_job_execution_params**, but it is empty for all the jobs.